data/extension/init/templates/tinygo/envoy.access_loggers/default/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxytest"
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/types"
+)
+
+func TestHelloWorld_OnTick(t *testing.T) {
+	opt := proxytest.NewEmulatorOption().
+		WithNewRootContext(newAccessLogger)
+	host := proxytest.NewHostEmulator(opt)
+	defer host.Done() // release the host emulation lock so that other test cases can insert their own host emulation
+
+	path := [2]string{":path", "/this/is/path"}
+	host.CallOnLogForAccessLogger([][2]string{path}, nil) // call OnLog
+
+	logs := host.GetLogs(types.LogLevelInfo)
+	require.Greater(t, len(logs), 0)
```

Comment:
[nit] Why do we need both styles of assertions -  `require.*` and  `assert.*` ?

Maybe, for the sake of simplicity, we could stick to one style ?